### PR TITLE
Fix sidebar navigation arrow

### PR DESCRIPTION
**Pull request by fvaldi**

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-1)

**Comment by fvaldi** [comment]
Hello Johannes, I'm having issues with this PR as I think that my project was created many time ago, however till today I wasn't able to work on it. I have copied the latest version from your main branch, but it seems that with the build step I'm having issues as it is not able to find the env variable 'NEXT_PUBLIC_API_BASE_URL'. Would it be possible to delete this repository so I can request from your website the creation of a new one that is up to date? Thanks in advance, Kind regards.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,32 +1,77 @@
...
+      - run: npm run lint:styles
       - run: npm run prettier
-      # verify that there are no type and build errors
+      - run: npm run check:types
+
+  build:
+    runs-on: ubuntu-latest
+    needs: install
+    env:
+      NEXT_PUBLIC_API_BASE_URL: https://prolog-api.profy.dev
```
Comment (jkettmann) [inline .github/workflows/main.yml]: This shouldn't be necessary anymore. I added this env variable to the repository settings. This should have been done automatically but sometimes it fails unfortunately

CleanShot 2023-10-24 at 08 50 52@2x

Comment (fvaldi) [reply .github/workflows/main.yml]: Thanks a lot Johannes for the fast response.

Kind regards.

**Comment by jkettmann** [comment]
Sorry, just saw that comment now. It looks like this is your first task. I'd recommend deleting the repo completely and creating it from scratch in the course material. This will get you to the latest version

**Comment by jkettmann** [comment]
Let me know if you'd like to delete the repo. You can try to delete it yourself (I just made you an admin) or reach out to me if that doesn't work.

**Comment by fvaldi** [comment]
Thank you, I think that I will keep the repo as I was able to update the code to the latest version from your repository. Also it seems that now the CICD is working after you added the var in the settings of the repository.

**Comment by jkettmann** [comment]
Alright. Just one more note: I'm currently preparing video solutions for all the tasks. Since the code base changed a bit (especially the migration from styled-components to SCSS modules) the solutions might look a bit different from your code. I don't have an ETA on when the videos will be ready yet but the first should come in the upcoming weeks

**Pull request merged**